server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixUserEntity.java

```diff
@@ -0,0 +1,28 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+import fr.uga.l3miage.tp1.exo2.enums.sexType;
+
+@Entity
+@Table(name="netflix_user")
+public class NetflixUserEntity {
+    @Id
+    private int id;
+
+    private String uuid;
+    private String lastname;
+    private String firstname;
+
+    @Enumerated(EnumType.ORDINAL)
+    private sexType sex;
```

Comment:
Et la précision de taille ? 
```suggestion
    @Enumerated(EnumType.ORDINAL)
    @Column(name="sex", length=2)
    private sexType sex;
```